cypress/e2e/facility_spec/FacilityHomepage.cy.ts

```diff
@@ -181,6 +221,99 @@ describe("Facility Homepage Function", () => {
     facilityNotify.closeNotificationSlide();
     loginPage.ensureLoggedIn();
     loginPage.clickSignOutBtn();
+    loginPage.loginManuallyAsDistrictAdmin();
+    loginPage.ensureLoggedIn();
+  });
+
+  it("Verify the bed capacity badge reflection", () => {
+    manageUserPage.typeFacilitySearch(facilityWithNoAvailableBeds);
+    facilityPage.verifyFacilityBadgeContent(facilityWithNoAvailableBeds);
+    manageUserPage.assertFacilityInCard(facilityWithNoAvailableBeds);
+    facilityHome.verifyOccupancyBadgeVisibility();
+    manageUserPage.assertFacilityBadgeContent("0", "0");
+
+    // link dummy doctor 12 to the facility
```

Comment:
```suggestion
    // link dummy doctor 12 to the facility
```

the test is about bed capacity badge reflection, why are you adding linking a new user to the facility in this test?

Reply:
That was for the whole patient/linking bed flow; but since you have mentioned it's not needed, removed 👍